packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -791,6 +792,23 @@ class FlutterDriver {
     return result;
   }
 
+  /// Force a garbage collection run in the VM.
+  Future<void> forceGC({ Duration timeout = _kShortTimeout }) async {
+    try {
+      await _peer
+          .sendRequest(_collectAllGarbageMethodName, <String, String>{
+            'isolateId': 'isolates/${_appIsolate.numberAsString}',
+          })
+          .timeout(timeout);
+    } catch (error, stackTrace) {
+      throw DriverError(
+        'Failed to force a GC due to remote error',
+        error,
+        stackTrace,
+      );
+    }
+  }
+
```

Comment:
remove the timeout logic (if you don't, i'll just rip it out momentarily anyway...)